src/CommonDBTM.php

```diff
@@ -1746,6 +1746,11 @@ protected function cleanLockedsOnAdd()
                 if ($lock == 'states_id' && $config['states_id_default']) {
                     continue;
                 }
+                //bypass for entities_id // default inventory conf is 0 'root entity')
+                if ($lock == 'entities_id') {
+                    continue;
+                }
```

Comment:
What should be done if there is a global lock on `entities_id` and at the same time a rule defines it to something else than `0`? Should we force the `0` value or should we keep the value defined by the rule?

Reply:
for me, a global lock (on the entity) at creation makes no sense
for creation, you don't have to deal with the case of the entity (which by default is at 0 and potentially redefined by the rules)

In any case, we have data

Reply:
I agree with Stan. Locking on add does not make sens, since nothing exists yet; so we'll take default configured entoity, or the one defined by rules. Locking (global or not btw) will make its job on update.